src/test_suite.js

```diff
@@ -12,6 +12,61 @@ TestSuite.prototype.name = function (name) {
   return this;
 };
 
+TestSuite.prototype.timestamp = function (timestamp) {
+  if(_.isDate(timestamp)) timestamp = timestamp.toISOString();
```

Comment:
I was working from example output from an alternative converter which included the milliseconds (but without the Z). I worked under the assumption that any ISO-8601 compliant format would suffice, but your schema indicates it's not that flexible. I'll modify it so that it conforms. Do you have any issue with adding a dependency of [dateformat](https://www.npmjs.com/package/dateformat)?